src/main/java/pl/touk/sputnik/connector/gerrit/GerritFacadeBuilder.java

```diff
@@ -22,11 +25,24 @@ public GerritFacade build() {
         ConnectorDetails connectorDetails = new ConnectorDetails();
         GerritPatchset gerritPatchset = buildGerritPatchset();
 
-        HttpHost httpHost = httpHelper.buildHttpHost(connectorDetails);
-        HttpClientContext httpClientContext = httpHelper.buildClientContext(httpHost, new DigestScheme());
-        CloseableHttpClient closeableHttpClient = httpHelper.buildClient(httpHost, connectorDetails);
-
-        return new GerritFacade(new GerritConnector(new HttpConnector(closeableHttpClient, httpClientContext, connectorDetails.getPath()), gerritPatchset));
+        GerritRestApiFactory gerritRestApiFactory = new GerritRestApiFactory();
+        String hostUri = httpHelper.buildHttpHostUri(connectorDetails);
+        if (!Strings.isNullOrEmpty(connectorDetails.getPath())) {
+            hostUri += connectorDetails.getPath();
+        }
+        log.info("Using Gerrit URL: {}", hostUri);
+        GerritAuthData.Basic authData = new GerritAuthData.Basic(hostUri,
```

Comment:
In the next version of gerrit-rest-java-client, there is improved support for some special auth methods like LDAP with HTML-form and HTTP digest included. I will create a PR when ready.

I have updated my PR with your suggested fixes.